Stripe/STPAPIClient.m

```diff
@@ -449,3 +463,55 @@ - (void)stopPollingSourceWithId:(NSString *)identifier {
 }
 
 @end
+
+#pragma mark - Customers
+
+@implementation STPAPIClient (Customers)
+
++ (STPAPIClient *)apiClientWithEphemeralKey:(STPEphemeralKey *)key {
+    STPAPIClient *client = [[self alloc] init];
+    client.apiKey = key.secret;
```

Comment:
I decided to make these ephemeral key auth'd requests class methods because I think it's safer, and simpler to explain in the future when we make these methods public. Since each request requires an ephemeral key, and creates a one-off API client, we don't have to worry about guarding against a nil key, or what happens if you change the client's ephemeral key while a request is in progress.

I think we can still add config options in the future pretty easily, by adding a config parameter to each request.